http-server/src/test/java/org/triplea/server/http/ClientServerIntegrationTest.java

```diff
@@ -0,0 +1,145 @@
+package org.triplea.server.http;
```

Comment:
This was existing, but renamed. This class is an integration test that sets up a server if not already running and exercises all http clients. I wanted to break it up per client, but the constant starting/stopping was slow and not reliable.